src/chunkserver/chunk_service.cpp

```diff
@@ -65,8 +78,8 @@ void ChunkServiceImpl::DeleteChunk(RpcController *controller,
                                                response,
                                                done);
     CHECK(nullptr != closure) << "new chunk service closure failed";
-
     brpc::ClosureGuard doneGuard(closure);
+    VERIFY_CREDENTIAL(request, response);
 
```

Comment:
it's better to clear token in request after succeeded verification, because the whole request will be serialized into WAL.

Reply:
Are you worried about safety issues? token is encrypted ciphertext and only valid for 15s

Reply:
but the request here is const object